### PR TITLE
Logi skill fix & seperation

### PR DESCRIPTION
**Pull request by testkil**

### CODE REVIEW

File: backend/data/skills.yaml [hunk 1]
```diff
@@ -241,16 +241,96 @@ requirements:
...
+      elite: 5
+    Nanite Interfacing:
+      elite: 4
+    Nanite Operation:
+      elite: 4
+    Target Management:
       min: 5
+      elite: 5
+    Long Range Targeting:
+      min: 3
```
Comment (TDF-Austar) [inline backend/data/skills.yaml:321]: min should be 5 as cannot fly cruiser logi without it, min req to fly nestor is 20 hours in cruiser, even if using Nestor only alt should still be 5 as directly affects targeting range.

File: backend/data/skills.yaml [hunk 2]
```diff
@@ -241,16 +241,96 @@ requirements:
...
+      elite: 5
+    Signature Analysis:
+      elite: 5
```
Comment (TDF-Austar) [inline backend/data/skills.yaml:324]: min should be 5 as cannot fly cruiser logi without it, directly affects lock speed of all ships, important for nestor, and as a pre req for cruiser should apply to nestor directly as well imo.

File: backend/data/skills.yaml [hunk 3]
```diff
@@ -424,23 +504,27 @@ requirements:
...
-      min: 3
+      min: 1
       elite: 5
+    CPU Management:
+      min: 4
+    Power Grid Management:
+      min: 5
     Cybernetics:
       min: 5
+    Long Range Targeting:
```
Comment (TDF-Austar) [inline backend/data/skills.yaml:523]: Skill can be removed from ship specific as should be an all logi requirement imo.

File: backend/data/skills.yaml [hunk 4]
```diff
@@ -450,24 +534,30 @@ requirements:
...
+      min: 4
+    Power Grid Management:
+      min: 5
     Frequency Modulation:
       elite: 4
     Long Distance Jamming:
       elite: 4
+    Drones:
+      min: 5
+    Long Range Targeting:
```
Comment (TDF-Austar) [inline backend/data/skills.yaml:556]: Skill can be removed from ship specific as should be an all logi requirement imo.

**Review by TDF-Austar — CHANGES_REQUESTED** [review]
Review of skill reqs proposed. See comments.

File: backend/data/skills.yaml (hunk 1 above)
Comment (testkil) [reply backend/data/skills.yaml:321]: guard/oni have min 5

**Pull request merged**